test/langtools/tools/javac/doctree/SeeTest.java

```diff
@@ -50,6 +50,24 @@ void quoted_text() { }
         Text[TEXT, pos:12, "String"]
     ]
 ]
+*/
+
+    /**
+     * Test '@' in quoted string.
+     * @see "{@code}"
+     */
+    void at_sign_in_quote_string() { }
```

Comment:
(minor) add `d` after `quote` in method name

Reply:
Also, add a test case for this, with a newline after the tag:
```

    /**
     * abc.
     * @see
     *    "{@code ...}"
     */
```

Reply:
> (minor) add `d` after `quote` in method name

Fixed.